readthedocs/builds/models.py

```diff
@@ -705,3 +708,117 @@ def run_time(self):
         if self.start_time is not None and self.end_time is not None:
             diff = self.end_time - self.start_time
             return diff.seconds
+
+
+class VersionAutomationRule(PolymorphicModel, TimeStampedModel):
+
+    """Versions automation rules for projects."""
+
+    ACTIVATE_VERSION_ACTION = 'activate-version'
+    ACTIONS = (
+        (ACTIVATE_VERSION_ACTION, _('Activate version on match')),
+    )
+
+    project = models.ForeignKey(
+        Project,
+        related_name='automation_rules',
+        on_delete=models.CASCADE,
+    )
+    priority = models.IntegerField(
```

Comment:
Are we using this somewhere?

I assume that this could be useful to decide what rule to apply first. If that's the case, maybe `order` is a better name for this.

Reply:
Yeah, this is to know the order of the rules.

Reply:
Yep, this defines which rule is run first. Not sure about order, it translates different without context?